crates/gpui/src/text_system/line_wrapper.rs

```diff
@@ -117,7 +117,7 @@ impl LineWrapper {
                 buffer,
                 self.font_size,
                 &[FontRun {
-                    len: 1,
+                    len: buffer.len(),
```

Comment:
If I understand `FontRun` correctly, this `len` should return the UTF-8 length of the character, for example, the Chinese character "我" should be 3 bytes instead of 1.

Reply:
https://github.com/zed-industries/zed/blob/6d5787cfdc53441edaba5b2c5d82a7aeafd53e1e/crates/gpui/src/platform/mac/text_system.rs#L451

Looks like it should be so.